docs/api-reference.md

```diff
@@ -1560,6 +1701,38 @@ Used in:
 * [PackageManifestSpec](#packagemanifestspec)
 
 
+### PackageManifestPlatformVersionConstraint
+
+PackageManifestPlatformVersionConstraint enforces that the platform matches the given version range.
+This constraint is ignored when running on a different platform.
+e.g. a PlatformVersionConstraint OpenShift>=4.13.x is ignored when installed on a plain Kubernetes cluster.
+Use the Platform constraint to enforce running on a specific platform.
+
+| Field | Description |
+| ----- | ----------- |
+| `name` <b>required</b><br><a href="#platformname">PlatformName</a> | Name of the platform this constraint should apply to. |
+| `range` <b>required</b><br>string | Semantic Versioning 2.0.0 version range. |
+
+
+Used in:
+* [PackageManifestConstraint](#packagemanifestconstraint)
+
+
+### PackageManifestRepository
+
+PackageManifestRepository contains information about one package repository
+which could be loaded either from a local file or from a container image.
+
+| Field | Description |
+| ----- | ----------- |
+| `file` <br>string | References a file in the filesystem to load. |
```

Comment:
Does this reference a file or a folder with the repository date inside?
Seems like `folder`

Reply:
Nope, the code right now loads a repository manifest from there by doing `open` on it